### PR TITLE
Test Data Credits

### PR DESCRIPTION
**Pull request by vihu**

### CODE REVIEW

File: config/sys.config [hunk 1]
```diff
@@ -73,7 +73,7 @@
    {election_interval, 30},
    {radio_device, { {127,0,0,1}, 1680,
                     {127,0,0,1}, 31341} },
-   {default_routers, ["/p2p/112qB3YaH5bZkCnKA5uRH7tBtGNv2Y5B4smv1jsmvGUzgKT71QpE", "/p2p/1124CJ9yJaHq4D6ugyPCDnSBzQik61C1BqD9VMh1vsUmjwt16HNB"]},
+   {default_routers, ["/p2p/112qB3YaH5bZkCnKA5uRH7tBtGNv2Y5B4smv1jsmvGUzgKT71QpE"]},
```
Comment (macpie) [inline config/sys.config:76]: idk if we should remove this just yet

File: test/miner_packet_routing_SUITE.erl [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    timer:sleep(5000),
+    true = load_genesis_block(GenesisBlock, Miners, Config),
+    miner_fake_radio_backplane ! go,
+    Config.
+
+end_per_testcase(TestCase, Config) ->
+    gen_server:stop(miner_fake_radio_backplane),
+    miner_ct_utils:end_per_testcase(TestCase, Config).
+
+basic_test(Config) ->
```
Comment (macpie) [inline test/miner_packet_routing_SUITE.erl:59]: What are we testing here?

File: test/miner_packet_routing_SUITE.erl [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    Miners = ?config(miners, Config),
+    ok = miner_ct_utils:wait_for_gte(height, Miners, 5),
+    miner_fake_radio_backplane:transmit(<<?JOIN_REQUEST:3, 0:5, 1337:64/integer-unsigned-big, 1234:64/integer-unsigned-big, 1111:16/integer-unsigned-big, 0:32/integer-unsigned-big>>, 911.200, 631210968910285823),
+    ok = miner_ct_utils:wait_for_gte(height, Miners, 10),
+    ok.
+
+
+get_genesis_block(Miners, Config) ->
```
Comment (macpie) [inline test/miner_packet_routing_SUITE.erl:67]: Maybe put this in test_utils? (and functions below)

File: test/miner_state_channel_SUITE.erl [hunk 4]
```diff
@@ -0,0 +1,546 @@
...
+    InitialDCTxns = [blockchain_txn_dc_coinbase_v1:new(Addr, Balance) || Addr <- Addresses],
+    AddGwTxns = [blockchain_txn_gen_gateway_v1:new(Addr, Addr, h3:from_geo({37.780586, -122.469470}, 13), 0)
+                 || Addr <- Addresses],
+
+    NumConsensusMembers = ?config(num_consensus_members, Config),
+    BlockTime = ?config(block_time, Config),
+    BatchSize = ?config(batch_size, Config),
+    Curve = ?config(dkg_curve, Config),
+
+    SCVars = #{max_open_sc => 2,
```
Comment (macpie) [inline test/miner_state_channel_SUITE.erl:54]: Comments on these vars so we know what they mean

File: test/miner_state_channel_SUITE.erl [hunk 5]
```diff
@@ -0,0 +1,546 @@
...
+    RouterPubkeyBin = libp2p_crypto:pubkey_to_bin(RouterPubkey),
+    RouterSwarm = ct_rpc:call(RouterNode, blockchain_swarm, swarm, []),
+    ct:pal("RouterSwarm: ~p", [RouterSwarm]),
+    RouterP2PAddress = ct_rpc:call(RouterNode, libp2p_swarm, p2p_address, [RouterSwarm]),
+    ct:pal("RouterP2PAddress: ~p", [RouterP2PAddress]),
+
+    ct:pal("Height before oui: ~p", [miner_ct_utils:height(RouterNode)]),
+
+    EUIs = [{16#deadbeef, 16#deadc0de}],
+    {Filter, _} = xor16:to_bin(xor16:new([ <<DevEUI:64/integer-unsigned-little,
```
Comment (macpie) [inline test/miner_state_channel_SUITE.erl:112]: Could you explain what the filter does here?

File: config/sys.config (hunk 1 above)
Comment (vihu) [reply config/sys.config]: Re-added

File: test/miner_packet_routing_SUITE.erl (hunk 3 above)
Comment (vihu) [reply test/miner_packet_routing_SUITE.erl]: Done

File: test/miner_state_channel_SUITE.erl (hunk 4 above)
Comment (vihu) [reply test/miner_state_channel_SUITE.erl]: Added

File: test/miner_state_channel_SUITE.erl (hunk 5 above)
Comment (vihu) [reply test/miner_state_channel_SUITE.erl:114]: Put a comment there

**Comment by Vagabond** [comment]
closing in favor of #389

**Pull request closed without merging**